citing-us/index.html

```diff
@@ -0,0 +1,88 @@
+                <p>The original OMERO paper from 2012:</p>
+                <p class="citation">Chris Allan, Jean-Marie Burel, Josh Moore, Colin Blackburn, Melissa Linkert, Scott Loynton, Donald MacDonald, William J Moore, Carlos Neves, Andrew Patterson, Michael Porter, Aleksandra Tarkowska, Brian Loranger, Jerome Avondo, Ingvar Lagerstedt, Luca Lianas, Simone Leo, Katherine Hands, Ron T Hay, Ardan Patwardhan, Christoph Best, Gerard J Kleywegt, Gianluigi Zanetti &amp; Jason R Swedlow (2012) OMERO: flexible, model-driven data management for experimental biology. Nature Methods 9, 245–253. Published: 28 February 2012</p>
+                <p><a class="pmid" href="http://www.ncbi.nlm.nih.gov/pubmed/22373911" target="_blank">22373911</a></p>
+                <p>OMERO for high content screening:</p>
+                <p class="citation">Simon Li, Sébastien Besson, Colin Blackburn, Mark Carroll, Richard K. Ferguson, Helen Flynn, Kenneth Gillen, Roger Leigh, Dominik Lindner, Melissa Linkert, William J. Moore, Balaji Ramalingam, Emil Rozbicki, Gabriella Rustici, Aleksandra Tarkowska, Petr Walczysko, Eleanor Williams, Chris Allan, Jean-Marie Burel, Josh Moore, Jason R. Swedlow (2016) Metadata management for high content screening in OMERO. Methods 96, 27-32. Published: 1 March 2016</p>
+                <p><a href="http://dx.doi.org/10.1016/j.ymeth.2015.10.006" target="_blank">doi:10.1016/j.ymeth.2015.10.006</a></p>
+                <p>Publishing and sharing with OMERO:</p>
+                <p class="citation">Jean-Marie Burel, Sébastien Besson, Colin Blackburn, Mark Carroll, Richard K. Ferguson, Helen Flynn, Kenneth Gillen, Roger Leigh, Simon Li, Dominik Lindner, Melissa Linkert, William J. Moore, Balaji Ramalingam, Emil Rozbicki, Aleksandra Tarkowska, Petr Walczysko, Chris Allan, Josh Moore, Jason R. Swedlow (2015) Publishing and sharing multi-dimensional image data with OMERO. Mammalian Genome 26, 441-447. Published 30 July 2015</p>
+                <p><a href="http://dx.doi.org/10.1007/s00335-015-9587-6" target="_blank">doi:10.1007/s00335-015-9587-6</a></p>
+            </div>            
+        </div>      
+        <!-- end citation-->
+        
+        <!-- begin citation--> 
+        <hr class="whitespace">
+        <div class="row">
+            <div class="small-12 medium-4 columns">
+                <h3>Image Data Resource (IDR)</h3>
+            </div>
+            <div class="small-12 medium-8 columns">
+                <p class="citation">Eleanor Williams, Josh Moore, Simon W. Li, Gabriella Rustici, Aleksandra Tarkowska, Anatole Chessel, Simone Leo, Bálint Antal, Richard K. Ferguson, Ugis Sarkans, Alvis Brazma, Rafael E. Carazo Salas, Jason R. Swedlow (2017) The Image Data Resource: A Bioimage Data Integration and Publication Platform. Nature Methods 14(8), 775-781. Published 19 June 2017</p>
+                <p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536224/" target="_blank">PubMedCentral: PMC5536224</a></p>
```

Comment:
Replace 'PubMedCentral:' with 'PMCID:'
(sorry wasn't sure if that was the convention but found an example now)